vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -82,6 +76,6 @@ cut_official_branch() {
 }
 
 set_git_config
-find_current_image_version "pkg/agent/datamodel/sig_config.go"
+find_current_image_version "linux_sig_version.json"
```

Comment:
@ganeshkumar5699 this should have been the absolute file path, its failing in vhd automation. it never got caught because automation never worked since this PR merged. I will fix it.